R/utilities-parameters.R

```diff
@@ -50,6 +50,10 @@ readParametersFromXLS <- function(paramsXLSpath, sheets = NULL) {
     }
   }
 
+  # replace `NA` in units with `""` represeting the empty unit for
+  # the dimension `Dimensionless`
+  units <- tidyr::replace_na(data = units, replace = "")
```

Comment:
The dimension "Dimensionless" has a unit `""` (empty string). When this unit is stored in excel and read in R, the value of the cell is `NA`. I am replacing here with the empty string so this becomes a valid unit.

Reply:
It can potentially be fraction FYI

Reply:
Yepp, thanks.